Tests/TuistGeneratorIntegrationTests/Utils/EmbedScriptGeneratorIntegrationTests.swift

```diff
@@ -45,4 +45,30 @@ final class EmbedScriptGeneratorIntegrationTests: TuistTestCase {
         XCTAssertTrue(got.script.contains("install_bcsymbolmap \"2510FE01-4D40-3956-BB71-857D3B2D9E73.bcsymbolmap\""))
         XCTAssertTrue(got.script.contains("install_bcsymbolmap \"773847A9-0D05-35AF-9865-94A9A670080B.bcsymbolmap\""))
     }
+
+    func test_script_when_not_includingSymbolsInFileLists() throws {
+        // Given
+        let carthagePath = try temporaryFixture("Carthage/")
+        let frameworkPath = FileHandler.shared.glob(carthagePath, glob: "*.framework").first!
+        let framework = FrameworkNode(path: frameworkPath)
+
+        // When
+        let got = try subject.script(sourceRootPath: carthagePath, frameworkPaths: [framework.path], includeSymbolsInFileLists: false)
+
+        // Then
+        XCTAssertFalse(got.inputPaths.contains(RelativePath("2510FE01-4D40-3956-BB71-857D3B2D9E73.bcsymbolmap")))
```

Comment:
minor: A simplification here could be

```swift
XCTAssertEqual(got.inputPaths, [
      RelativePath("RxBlocking.framework")
])
        
XCTAssertEqual(got.outputPaths, [
      "${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/RxBlocking.framework"
 ])
```

This ensures the test continues to be valid (even if the underlying UUIDs change in the future)

Reply:
Makes a lot of sense 👍